yr.py

```diff
@@ -176,7 +176,7 @@ def __init__(self, filename):
 
     def _process_row(self, row):
         trans = Transaction()
-        trans.date = datetime.strptime(row["Date"], "%Y/%m/%d")
```

Comment:
If this line was a problem for you, I'll bet YNAB is honoring locale on data export. So rather than changing the date so it works for you but not me, I'd rather either make this configurable.